From 8256a06230c9b6b1fca6eeadc249c19953a90242 Mon Sep 17 00:00:00 2001
From: Daniel Muey <dan@cpanel.net>
Date: Tue, 28 Jan 2020 17:45:17 -0600
Subject: [PATCH 01/11] EA4 OBS ready

1. Remove faulty libxml2 check
2. Remove faulty systemd check from fpm sapi config
---
 build/php.m4       | 2 --
 sapi/fpm/config.m4 | 2 --
 2 files changed, 4 deletions(-)

diff --git a/build/php.m4 b/build/php.m4
index 7392876..3b0917c 100644
--- a/build/php.m4
+++ b/build/php.m4
@@ -2026,8 +2026,6 @@ dnl
 dnl Common setup macro for libxml.
 dnl
 AC_DEFUN([PHP_SETUP_LIBXML], [
-  PKG_CHECK_MODULES([LIBXML], [libxml-2.0 >= 2.7.6])
-
   PHP_EVAL_INCLINE($LIBXML_CFLAGS)
   PHP_EVAL_LIBLINE($LIBXML_LIBS, $1)
 
diff --git a/sapi/fpm/config.m4 b/sapi/fpm/config.m4
index ad89c1b..fc06a05 100644
--- a/sapi/fpm/config.m4
+++ b/sapi/fpm/config.m4
@@ -549,8 +549,6 @@ if test "$PHP_FPM" != "no"; then
     [no])
 
   if test "$PHP_FPM_SYSTEMD" != "no" ; then
-    PKG_CHECK_MODULES([SYSTEMD], [libsystemd >= 209])
-
     AC_CHECK_HEADERS(systemd/sd-daemon.h, [HAVE_SD_DAEMON_H="yes"], [HAVE_SD_DAEMON_H="no"])
     if test $HAVE_SD_DAEMON_H = "no"; then
       AC_MSG_ERROR([Your system does not support systemd.])
-- 
2.24.1

